ui/vite.config.js

```diff
@@ -4,10 +4,44 @@ import { defineConfig } from "vite";
 import vue from "@vitejs/plugin-vue";
 import monacoEditorPlugin from "vite-plugin-monaco-editor";
 
```

Comment:
Can you please look into having the Vite config as TypeScript? Pretty sure a `vite.config.ts` is possible.

I'm not sure why I created a JS config.